assets/src/utils/parse-html-placeholders.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External imports
+ */
+import { isEmpty } from 'lodash';
+
+const PLACEHOLDER = '%%var%%';
+
+/**
+ * kinda like a weaker version of sprintf that won't choke on HTML elements
+ *
+ * @function
+ * @param {string} placeholderText
+ * @param {Array} replacements
+ * @return {string} text with placeholders replaced by variables
+ */
+const parseHtmlPlaceholders = ( placeholderText, replacements ) => {
+	if ( isEmpty( replacements ) ) {
+		return placeholderText;
+	}
+	if ( ! Array.isArray( replacements ) ) {
+		replacements = [ replacements ];
+	}
+	if ( ! placeholderText.includes( PLACEHOLDER ) ) {
+		throw new RangeError(
```

Comment:
Why should there be an error if there are no placeholders? The default behavior of `sprintf` is that it returns the string if no placeholders are found. I think we should keep the behavior same as that.

That way, we can move this condition inside the first one:
```JS
if ( isEmpty( replacements ) || ! placeholderText.includes( PLACEHOLDER ) ) {
	return placeholderText;
}
```

Reply:
hmmm... ya good point... will change